gamemodes/cinema/gamemode/modules/theater/services/sh_dlive.lua

```diff
@@ -0,0 +1,82 @@
+-- This file is subject to copyright - contact swampservers@gmail.com for more information.
+
+local SERVICE = {}
+
+SERVICE.Name 		= "DLive"
+
+SERVICE.NeedsCodecs = true
+
+function SERVICE:GetKey(url)
+	if url.host and string.match(url.host,"dlive.tv") and string.match(url.path,"^/([%w_]+)[/]?$") then
+		return url.encoded
+	end
+	return false
+end
+
+if CLIENT then
+	function SERVICE:GetVideoInfoClientside(key, callback)
```

Comment:
It's probably possible to get the title with serverside HTTP, but I don't see it anywhere in the initial HTML page so this is fine